nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/test/java/org/nuxeo/ecm/automation/core/operations/blob/BlobToFileTest.java

```diff
@@ -53,12 +55,10 @@
 
     protected static final String DEFAULT_USER_ID = "hsimpsons";
 
-    protected static final String DIRECTORY = FileUtils.getTempDirectoryPath() + "/blob-to-file";
+    protected static final String DIRECTORY_PATH = FileUtils.getTempDirectoryPath() + "/blob-to-file";
```

Comment:
Don't use the (effectively fixed) temp dir + a fixed dir name. Temporary stuff should always be random to avoid a certain class of attacks that predicts you're going to write them. Use a `File directory` field and do in the `setUp`: `directory = Framework.createTempDirectory("blob-to-file").toFile();`